src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -264,6 +274,17 @@ const PermissionIdentities: FC = () => {
                 />
               )}
             </PageHeader.Left>
+            <PageHeader.BaseActions>
+              <Button
+                appearance="positive"
+                className="u-float-right u-no-margin--bottom"
+                onClick={openPortal}
+                hasIcon={!isSmallScreen}
+              >
+                {!isSmallScreen && <Icon name="plus" light />}
+                <span>Create TLS Identity</span>
+              </Button>
+            </PageHeader.BaseActions>
```

Comment:
All of this logic, with the button, mounting the modal and referencing the new CreateIdentityModal should move to a CreateTLSIdentityButton component, so we have contained the logic in a single place. This PermissionIdentities component responsibility should be the listing, not the action details.